packages/@aws-cdk/aws-ec2/test/integ.import-default-vpc.lit.ts

```diff
@@ -0,0 +1,24 @@
+import cdk = require('@aws-cdk/cdk');
+import ec2 = require("../lib");
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-ec2-import');
+
+/// !show
+const vpc = ec2.VpcNetworkRef.importFromContext(stack, 'VPC', {
```

Comment:
Maybe add some sugar methods for common use cases?

```ts
ec2.VpcNetwork.importDefaultVpc(stack, 'Vpc');
ec2.VpcNetwork.importByTag(stack, 'Vpc', { tags: { key: 'key', value: 'value' } });
ec2.VpcNetwork.importByName(stack, 'Vpc', 'vpcName');
```

What about VPC ID? (is that the same as name?)

Reply:
vpc-id is probably the most common. There is still a big value prop to using this because you are going to get the subnet-ids for free.

Reply:
The downside of vpc-id is that you 

a) Have to look it up manually and pass it into your CDK app somehow
b) It will be different for every region you plan to deploy to.

Both of these make it so that I think the more common use will be to do it by tag.

Reply:
vpcId is supported, and since we basically leak the VPC query API, I am okay with not adding the sugar methods as well.